core/domain/platform_parameter_list.py

```diff
@@ -61,3 +62,8 @@ class ParamNames(enum.Enum):
     'This flag is for the checkpoint celebration feature.',
     platform_parameter_domain.FeatureStages.PROD,
 )
+
+Registry.create_feature_flag(
+    ParamNames.ANDROID_PAGE,
+    'This flag is for the android page.',
```

Comment:
Perhaps it's worth being a bit clearer here on what is meant by 'the android page'? Given the entire site works on Android, it'd probably be better to update this to indicate that it gates the Android beta promo landing page.

Ditto for the page name.

Reply:
Done